packages/client/modules/email/components/EmailNotifications/EmailResponseMentioned.tsx

```diff
@@ -0,0 +1,55 @@
+import graphql from 'babel-plugin-relay/macro'
+import {EmailResponseMentioned_notification$key} from 'parabol-client/__generated__/EmailResponseMentioned_notification.graphql'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import makeAppURL from '../../../../utils/makeAppURL'
+import {notificationSummaryUrlParams} from '../NotificationSummaryEmail'
+import NotificationTemplate from './EmailNotificationTemplate'
+
+interface Props {
+  notificationRef: EmailResponseMentioned_notification$key
+  appOrigin: string
+}
+
+const ResponseMentioned = (props: Props) => {
```

Comment:
+1 from the component name I'm not sure I can fully understand when this email will be sent. Is there any way to make that clearer? Is it sent every time someone mentions me in the task or discussion thread?

Reply:
This is for when someone mentions you in a standup response.

Agree that the name is somewhat ambiguous, but I'd prefer to keep this as `EmailResponseMentioned` here to maintain consistency with the existing `ResponseMentioned` notification type + in-app component